### PR TITLE
apply responsive tiles to alarm, calendar, news, timer, mediation

### PR DESCRIPTION
**Pull request by hiroaki404**
I applied responsive tiles to alarm, calendar, news, timer, meditation.

image

image

image

image

image

### CODE REVIEW

File: WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/golden/GoldenTilesPreviewsRow3.kt [hunk 1]
```diff
@@ -61,6 +62,7 @@ fun News(context: Context) = TilePreviewData {
             it.deviceConfiguration,
             headline = "Millions still without power as new storm moves across US",
             newsVendor = "The New York Times",
+            date = LocalDate.now().minusDays(1),
```
Comment (yschimke) [inline WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/golden/GoldenTilesPreviewsRow3.kt:65]: I can see the appeal of both options, but is it worth using a fixed date for previews.

Particularly given future screenshot testing of previews?

**Review by yschimke — APPROVED** [review]

File: WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/golden/GoldenTilesPreviewsRow3.kt (hunk 1 above)
Comment (hiroaki404) [reply WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/golden/GoldenTilesPreviewsRow3.kt]: @yschimke 
I think you are exactly right!
I want to display yesterday, I changed it using a fixed Clock.

image

**Comment by yschimke** [comment]
Nit: the calendar preview shows a layout issue. But might be ok to demonstrate how reviews can highlight problems and solve later.

**Comment by hiroaki404** [comment]
I wonder this is a bug in the library.
I also see this phenomenon in the favorite contacts tile built into wear os.
When the text size is changed, the text overflows.

**Comment by yschimke** [comment]
Let's land as is and follow up

**Pull request merged**